tracer/src/Datadog.Trace/LibDatadog/HandsOffConfiguration/ConfiguratorHelper.cs

```diff
@@ -0,0 +1,117 @@
+// <copyright file="ConfiguratorHelper.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Datadog.Trace.LibDatadog.HandsOffConfiguration.InteropStructs;
+
+namespace Datadog.Trace.LibDatadog.HandsOffConfiguration;
+
+internal struct ConfiguratorHelper
+{
+    internal static ConfigurationResult GetConfiguration(bool debugEnabled, string? handsOffLocalConfigPath, string? handsOffFleetConfigPath, bool? isLibdatadogAvailable = null)
+    {
+        if (isLibdatadogAvailable is false)
+        {
+            return new ConfigurationResult(null, "Skipping hands-off configuration: as LibDatadog is not available", Result.LibDatadogUnavailable);
```

Comment:
```suggestion
            return new ConfigurationResult(null, "Skipping hands-off configuration because LibDatadog is not available", Result.LibDatadogUnavailable);
```